strip-coverletter-docker.sh

```diff
@@ -1,24 +1,32 @@
 #!/bin/bash
 set -eu
 
-in=$(realpath $1)
-bname_in=${in##*/} # /foo/bar/baz.pdf = baz.pdf
+in=$(realpath "$1")
+bname_in=${in##*/} # "/foo/bar/baz.pdf" => "baz.pdf"
 
 out=$2
-out_dir=$(dirname $out)
+out_dir=$(dirname "$out")
 bname_out=${out##*/}
 
-work_dir="/tmp/decap"
-logfile="$work_dir/$bname_in.log" # ll: /tmp/decap/baz.pdf.log
+# lsh@2022-08-02: default to the external volume mounted at /bot-tmp, if it exists.
+# this is easier than modifying elife-bot code.
+tmp_dir="/tmp"
+if [ -e "/bot-tmp" ]; then
+    tmp_dir="/bot-tmp/tmp"
```

Comment:
Now looking at this after approving the PR in `elife-bot-formula`, I'm not sure the `tmp` folder can be relied on to be here. You may consider just using `/bot-tmp` and then the decap files would presumably end up in `/bot-tmp/decap` folder.

Reply:
that's a good point, and `/bot-tmp/tmp/decap` is convoluted.